loginsightwebhookdemo/template.py

```diff
@@ -51,11 +69,43 @@ def template(ALERTID=None, TOKEN=None):
 
     # Defaults to Content-type: application/json
     # If changed you must specify the content-type manually
-    #headers = {'Content-type': 'application/json', 'Accept': 'application/json'}
-    headers = ''
+    headers = {'Content-type': 'application/json', 'Accept': 'application/json'}
+
+    ########################
+    #Fire and Forgot Systems
+    ########################
 
     if headers:
-        return sendevent(TEMPLATEURL, json.dumps(payload), headers)
+        return callapi(TEMPLATEURL, 'post', json.dumps(payload), headers)
     else:
-        return sendevent(TEMPLATEURL, json.dumps(payload))
-    #return sendevent(TEMPLATEURL, json.dumps(payload), headers, (TEMPLATEUSER, TEMPLATEPASS))
+        return callapi(TEMPLATEURL, 'post', json.dumps(payload))
+    #return callapi(TEMPLATEURL, 'post', json.dumps(payload), headers, (TEMPLATEUSER, TEMPLATEPASS))
+
+    #####################################
+    #Incident / Ticket Management Systems
+    #####################################
+
+    ## Get the list of open incidents that contain the AlertName from the incoming webhook
+    #incident = callapi(TEMPLATEURL + '/api/v2/search.json?query=type:ticket status:open subject:"' + a['AlertName'] + '"', 'get', None, headers, (USER, PASS))
+    #i = json.loads(incident)
+
+    #try: # Determine if there is an open incident already
```

Comment:
It is similar -- the goal of the template is to provide ways of implementing a new integration. Previously, we only had one way to provide integration (fire and forget), now we also support check, fire and forget so I added an example of that to the template (the goal is you use one or the other method and I assume fire and forget would be most common).